<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RegistrationDesk</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>4</y>
      <width>258</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="messageBoard"/>
     </item>
     <item>
      <widget class="QLineEdit" name="broadcastMessage"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>4</y>
      <width>651</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QListView" name="log"/>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="Form">
        <attribute name="title">
         <string>Form</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>0</y>
           <width>640</width>
           <height>410</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>114</x>
            <y>350</y>
            <width>110</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0.00</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>84</x>
            <y>350</y>
            <width>30</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string>â‚¹</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>84</x>
            <y>10</y>
            <width>471</width>
            <height>331</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_1">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nameEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>College:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QComboBox" name="collegeCombo"/>
             </item>
             <item>
              <widget class="QLineEdit" name="collegeEdit">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>E-mail:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="emailEdit">
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>32</number>
             </property>
             <property name="placeholderText">
              <string>someone@example.com</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Mobile:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="phoneEdit">
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="placeholderText">
              <string>9876543210</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Event:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QListWidget" name="eventList">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="resizeMode">
              <enum>QListView::Adjust</enum>
             </property>
             <property name="batchSize">
              <number>150</number>
             </property>
             <property name="selectionRectVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>350</y>
            <width>75</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <pointsize>13</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>350</y>
            <width>75</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <pointsize>13</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="EventDetails">
        <attribute name="title">
         <string>Event Details</string>
        </attribute>
        <widget class="QComboBox" name="eventCombo">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>80</y>
           <width>170</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>80</y>
           <width>80</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Event Name:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>110</y>
           <width>80</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Coordinator:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>157</x>
           <y>162</y>
           <width>80</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Volunteers:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>268</y>
           <width>80</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>174</x>
           <y>215</y>
           <width>80</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Contact:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>188</x>
           <y>320</y>
           <width>80</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Time:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>110</y>
           <width>200</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>140</y>
           <width>200</width>
           <height>66</height>
          </rect>
         </property>
        </widget>
        <widget class="QListWidget" name="listWidget_2">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>246</y>
           <width>200</width>
           <height>66</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>215</y>
           <width>200</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>320</y>
           <width>200</width>
           <height>22</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>401</x>
           <y>79</y>
           <width>30</width>
           <height>24</height>
          </rect>
         </property>
         <property name="text">
          <string>Go</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="Schedule">
        <attribute name="title">
         <string>Schedule</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>eventCombo</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>listWidget_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>log</tabstop>
  <tabstop>messageBoard</tabstop>
  <tabstop>broadcastMessage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>broadcastMessage</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>broadcastMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>broadcastMessage()</slot>
  <slot>otherCheck(QString)</slot>
  <slot>clearFields()</slot>
 </slots>
</ui>
